hack/lib/etcd.sh

```diff
@@ -27,9 +27,9 @@ kube::etcd::validate() {
     exit 1
   }
 
-  # validate it is not running
-  if pgrep -x etcd >/dev/null 2>&1; then
-    kube::log::usage "etcd appears to already be running on this machine (`pgrep -xl etcd`) (or its a zombie and you need to kill its parent)."
+  # validate etcd port is free
+  if netstat -nlt | grep ':'$ETCD_PORT' ' >/dev/null 2>&1; then
+    kube::log::usage "etcd appears to already be running on this machine (`netstat -nlt | grep ':'$ETCD_PORT' '`) (or its a zombie and you need to kill its parent)."
```

Comment:
We should also change this message, as we don't care if etcd is running, so much that we care if the etcd port is unexpectedly in use.

Reply:
updated the message